.gitignore

```diff
@@ -16,3 +16,4 @@
 /libpeerconnection.log
 npm-debug.log
 testem.log
+.vscode
```

Comment:
I try to avoid adding editor specific files to the repo, also if you add a global gitignore for this in your local system setup it will be a bit less hassle for you in **all** the repos you work in. 😃 

https://docs.github.com/en/free-pro-team@latest/github/using-git/ignoring-files?algolia-query=gitignore#configuring-ignored-files-for-all-repositories-on-your-computer